presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragmenter.java

```diff
@@ -596,6 +599,36 @@ public GroupedExecutionProperties visitAggregation(AggregationNode node, Void co
             return GroupedExecutionProperties.notCapable();
         }
 
+        @Override
+        public GroupedExecutionProperties visitWindow(WindowNode node, Void context)
+        {
+            return visitWindowFunctionNodeInternal(node);
+        }
+
+        @Override
+        public GroupedExecutionProperties visitRowNumber(RowNumberNode node, Void context)
+        {
+            return visitWindowFunctionNodeInternal(node);
+        }
+
+        @Override
+        public GroupedExecutionProperties visitTopNRowNumber(TopNRowNumberNode node, Void context)
+        {
+            return visitWindowFunctionNodeInternal(node);
+        }
+
+        private GroupedExecutionProperties visitWindowFunctionNodeInternal(PlanNode node)
+        {
+            checkState(
+                    node instanceof WindowNode || node instanceof RowNumberNode || node instanceof TopNRowNumberNode,
+                    "Expects only WindowNode, RowNumberNode or TopNRowNumberNode");
+            GroupedExecutionProperties properties = node.getSources().get(0).accept(this, null);
+            if (properties.isCurrentNodeCapable()) {
+                return new GroupedExecutionProperties(true, true, properties.capableTableScanNodes);
```

Comment:
Use the getter rather than accessing the private field directly.

Reply:
I found the usage of getter is not consistent in this class. Would you suggest a refactor commit to make them consistent before the current commit?